Makefile

```diff
@@ -0,0 +1,61 @@
+DC_RUN = \
+	docker-compose \
```

Comment:
I think `docker-compose` is effectively [deprecated](https://www.docker.com/blog/new-docker-compose-v2-and-v1-deprecation/). What do you think about using `docker compose` instead?

Reply:
Ah, yes thanks for reminding me. Fixed in 010e2966224da69d69416a790788cebe14d9dcf4.